devoutils/faker/scripts/faker_cli.py

```diff
@@ -32,21 +33,30 @@ def cli():
 @click.option('--address', help='address to send.')
 @click.option('--port', help='Port to send.')
 @click.option('--tag', help='Tag from Devo.')
-@click.option('--simulation', is_flag=True, help='Set as simulation.')
+@click.option('--file_name', help='File name to store events. If file name '
+              'exist will only store the events in a file. Can be used '
+              'with batch mode to set the file where store the batch '
+              'events')
+@click.option('--simulation', is_flag=True,
+              help='Set as simulation. Shows the event in the console, '
+                   'but do not send it')
 @click.option('--template', '-t', type=click.File('r'), required=True,
               help='Template to send.')
+@click.option('--providers', '-p', type=click.File('r'), required=False,
+              help='File with custom providers dict.')
 @click.option('--interactive', '-i', is_flag=True,
               help='Interactive mode.')
 @click.option('--raw_mode', '-raw', is_flag=True,
               help='Send raw mode.')
-@click.option('--prob', default=100, help='Probability (0-100).')
-@click.option('--freq', default="1-1", help='Frequency in seconds. Example:'
-                                            '"1.0-5.0" = random time '
-                                            'between 1 sec. to 5secs.')
+@click.option('--probability', default=100, help='Probability (0-100).')
+@click.option('--frequency', default="1-1", help='Frequency in seconds. '
```

Comment:
I think that the 99% of the use of the faker is with cli. I haven't seen the option to use an "Complex probability and frequency", If there is no option for this I think that we must add it

Reply:
I put it in all function but the documentation say, if hard to put complex frequency and probability in --flags, soo, I only put if you add it in the config.json/.yaml, not a a flag. If you have one idea about how we can manage in a shell the complex probability and frequency, send a message please!